### PR TITLE
chore(metadata): add more tags to `rotate-ccw` icon metadata

### LINKED ISSUE #2602: Add "replay" tag to `rotate-ccw`
### Package

- [X] site

### Description

The `rotate-ccw` icon should also have the "replay" tag added.

### Use cases

Better searching

### Checklist

- [X] I have searched the existing issues to make sure this bug has not already been reported.

### PR DESCRIPTION
**Pull request by chessurisme**
This close #2602

## What is the purpose of this pull request?

- [ ] New Icon
- [ ] Bug fix
- [ ] New Feature
- [x] Documentation update
- [ ] Other:

### Description

Add more tags to the `rotate-ccw`

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Review by jguddas — APPROVED** [review]

**Review by realguse — CHANGES_REQUESTED** [review]
Could you add `rewind` and `reverse`

File: icons/rotate-ccw.json [hunk 1]
```diff
@@ -17,7 +18,10 @@
     "rerun",
     "refresh",
     "backup",
-    "undo"
+    "undo",
+    "replay",
+    "redo",
+    "retry"
```
Comment (chessurisme) [inline icons/rotate-ccw.json:24]:
```suggestion
    "retry",
    "rewind",
    "reverse"
```

File: icons/rotate-ccw.json [hunk 2]
```diff
@@ -6,7 +6,9 @@
     "ericfennis",
     "danielbayley",
     "jguddas",
-    "karsa-mistmere"
+    "karsa-mistmere",
+    "chessurisme",
+    "realguse"
```
Comment (ericfennis) [inline icons/rotate-ccw.json:11]: Only contributors to the design of the icons should be placed here.

Comment (chessurisme) [reply icons/rotate-ccw.json:11]: well fixed that

**Review by ericfennis — APPROVED** [review]

**Pull request merged**